rs/nns/governance/src/neuron/types.rs

```diff
@@ -164,6 +141,17 @@ impl Neuron {
         self.dissolve_state_and_age
     }
 
+    /// When we turn on enforcement of private neurons, this will only return
+    /// Public or Private, not None. When that happens, we should define another
+    /// Visibility that does NOT have Unspecified.
+    pub fn visibility(&self) -> Option<Visibility> {
```

Comment:
NCR.

Have you considered making it return `Visibility` while treating `Unspecified` as `Private`? More generally, now this can return 4 different values: `Some(Private)`, `Some(Public)`, `Some(Unspecified)`, and `None`. Do we need all 4, or can we have 3 or even 2?

Reply:
None will be returned during the migration window (to indicate that we are in the migration window).

I agree that it is suboptimal that the signature currently says that Unspecified might be returned. FWIW, the comment says we will fix that later. I just now created a ticket to make sure I don't forget: https://dfinity.atlassian.net/browse/NNS1-3239